server/routes/termdb.cluster.ts

```diff
@@ -77,7 +83,9 @@ async function getResult(q: TermdbClusterRequest, ds: any) {
 		_q.forClusteringAnalysis = true
 	}
 
-	const { term2sample2value, byTermId, bySampleId } = await ds.queries[q.dataType].get(_q) // too strong assumption on queries[dt], may not work for single cell
+	const { term2sample2value, byTermId, bySampleId } =
```

Comment:
please change to if-else. easier to expand later
if(q.dataType='numericDictterm') {
  getNumericDictTermAnnotation()
} else {
  await ds.queries[q.dataType]?.get(_q)
}